pmd-ui/src/main/resources/net/sourceforge/pmd/util/fxdesigner/fxml/designer.fxml

```diff
@@ -63,14 +63,6 @@
                                     <bottom>
                                         <ToolBar id="main-toolbar" prefHeight="40.0" prefWidth="200.0" BorderPane.alignment="CENTER">
                                             <items>
-                                                <Button fx:id="refreshASTButton" maxHeight="22.0" minHeight="22.0" mnemonicParsing="true" prefHeight="20.0" text="_Refresh AST">
```

Comment:
I think we could now have a tiny title label before each choicebox in the toolbar, eg "Language version:" and "XPath version:". Try how that looks and keep it if it's pretty